wp-content/themes/csisnuclearnetwork/assets/_scss/abstracts/_variables.scss

```diff
@@ -9,6 +9,7 @@ $color-white-130: rgba(255, 255, 255, 0.3);
 $color-white-140: rgba(255, 255, 255, 0.65);
 $color-white-150: rgba(255, 255, 255, 0.5);
 $color-white-165: rgba(255, 255, 255, 0.65);
+$color-text-white-180: rgba(255, 255, 255, 0.8);
```

Comment:
I'd recommend changing this variable name to `$color-white-185`. The `text` bit isn't really necessary (I know that comes from Figma).